src/main/java/com/gildedrose/AgeableController.java

```diff
@@ -0,0 +1,48 @@
+package com.gildedrose;
+
+import com.gildedrose.items.Ageable;
+import waiter.Request;
+import waiter.Response;
+import waiter.ResponseBuilder;
+
+import java.util.function.Function;
+
+public class AgeableController {
+
+    private final DataRepository ageableRepository;
+
+    public AgeableController() {
+        this.ageableRepository = new AgeableRepository();
+    }
+
+    final Function<Request, Response> okAllItemsHandler = request -> new ResponseBuilder()
+            .newUp()
+            .body(getJsonFormattedBody())
+            .headers(Response.HeaderField.ContentType, "application/json")
+            .build();
```

Comment:
Nice use of the Builder pattern! 👏

Reply:
Thank you! And thank you for all the review/comments! 😄